src/main/java/org/cryptomator/ui/addvaultwizard/CreateNewVaultLocationController.java

```diff
@@ -96,6 +100,20 @@ public class CreateNewVaultLocationController implements FxController {
 		this.usePresetPath = new SimpleBooleanProperty();
 		this.radioButtons = FXCollections.observableArrayList();
 		this.sortedRadioButtons = radioButtons.sorted(this::compareLocationPresets);
+		this.settings = settings;
+
+		var previouslyUsedDirectory = settings.previouslyUsedVaultDirectory.get();
+		if (previouslyUsedDirectory != null) {
+			try {
+				Path cachedPath = Paths.get(previouslyUsedDirectory);
```

Comment:
Don't use `Paths.get(...)". Use [Path::of](https://docs.oracle.com/en/java/javase/24/docs/api/java.base/java/nio/file/Path.html#of(java.lang.String,java.lang.String...)) instead.